### PR TITLE
Gör om första sidan

### PR DESCRIPTION
**Pull request by saeraa**
Redone the index page to use Next. Carousel looks a bit different than before, I used a library for simplicity. 
Some of the link href's are probably not correct, I realise we haven't discussed those and should do so soonish.

### CODE REVIEW

**Comment by Arash-SJPI22** [comment]
Nice work!
Have you thought about using <Link instead of <a to utilize Next.js more ?

**Comment by saeraa** [comment]
> 

Yes, excellent suggestion. I just copy-pasted a lot. Fixed the booking links to match the ones from the individual pages as well.

File: src/components/showtimesList.js [hunk 1]
```diff
@@ -1,14 +1,15 @@
 import ShowtimesListDay from "./showtimesListDay";
 import styles from "@/styles/asideShowTimes.module.scss";
 
-const showtimesList = ({ showtimes }) => {
+const showtimesList = ({ showtimes, movies }) => {
```
Comment (saeraa) [inline src/components/showtimesList.js:4]: There's some prop drilling here which I will change if I have time.. for now, this works!

**Review by davidlidebrandt — APPROVED** [review]
I think it looks great and the code looks clean and well organized, no remarks.

**Pull request merged**